### ISSUE TITLE
SD-Card support

### ISSUE THREAD

**Issue by UnknownAMD** [UpsilonNumworks/Upsilon#325]
#**Is your feature request related to a problem? Please describe.**
58KB (or the 8mb of flash) is not enough for my python scripts i plan to solder a slot to my numworks on the existing sdcard pads

**Describe the solution you'd like**
Since this would actually not only help me but a ton of hackers what i'd like is adding python IO support for the SD card slot and also some Ion api functions to allow people to make better games, i currently need this because im working on a video player but i dont have enough space for more than 5 minutes of MP4

**Additional context**
N/A

**Summary**
- Python IO support for the SD card
- Ion functions for the SD card READ/WRITE and more
- simple FS preferably FAT or a custom one (This is what takes the most time)
- usable as boot device (optional)

**Comment by Alerymin** [comment]
Actually, Python scripts are stored in RAM (256 Kio on n0100 and n0110, 564 Kio on n0120)
As far as I know, only the n0100 has SD card pads, and this model only has 1MB for the system.
It would be interesting but really complicated, [Lambda](https://github.com/Lambda-Numworks/Lambda) is meant to add features like this, but it would be really complicated

**Comment by UnknownAMD** [comment]
i have inspected the pcb and i have concluded that it would not require any more than 6 jumpers

**Comment by UnknownAMD** [comment]
from the sd slot to the board

**Comment by UnknownAMD** [comment]
a mod board of some sort could be designed or a new pcb (this would require moving all components)

**Comment by Alerymin** [comment]
Yes, but you don't know how complicated it is to add compatibiity with this sd card slot

**Comment by UnknownAMD** [comment]
numworks devs said there are ion functions that do basic read/write

**Comment by UnknownAMD** [comment]
to sdcard

**Comment by BloomyInDev** [comment]
Any news on this ?

**Comment by Lisra-git** [comment]
Absolutely useless

**Comment by BloomyInDev** [comment]
I dont think, having an sdcard to store big script/external app would be cool
